packages/spor-react/src/theme/components/checkbox.ts

```diff
@@ -1,86 +1,61 @@
-import { checkboxAnatomy as parts } from "@chakra-ui/anatomy";
-import { createMultiStyleConfigHelpers } from "@chakra-ui/react";
-import { mode } from "@chakra-ui/theme-tools";
-import { baseBackground, baseText } from "../utils/base-utils";
-import { brandBackground, brandText } from "../utils/brand-utils";
+import { brandBackground } from "../utils/brand-utils";
 import { focusVisibleStyles } from "../utils/focus-utils";
+import { defineRecipe } from "@chakra-ui/react";
 
-const helpers = createMultiStyleConfigHelpers(parts.keys);
-
-const config = helpers.defineMultiStyleConfig({
-  baseStyle: (props) => ({
-    container: {
-      _hover: {
-        "input:enabled:not([aria-invalid]) + .chakra-checkbox__control": {
-          ...baseBackground("hover", props),
-          borderColor: brandBackground("hover", props).backgroundColor,
-        },
-        "input:enabled[aria-invalid] + .chakra-checkbox__control": {
-          backgroundColor: mode("white", "inherit")(props),
-          borderColor: mode("outline.error.light", "outline.error.dark"),
+export const checkboxRecipe = defineRecipe({
```

Comment:
Her må du bruke defineSlotRecipe. Denne komponenten virker å ha flere slots definert. Se drawer.ts i theme/components mappen for referanse